app/react/App/Menu.js

```diff
@@ -1,38 +1,37 @@
 import PropTypes from 'prop-types';
-import React, {Component} from 'react';
-import {connect} from 'react-redux';
-import {NeedAuthorization} from 'app/Auth';
-import {I18NLink, I18NMenu, t} from 'app/I18N';
-import {processFilters, encodeSearch} from 'app/Library/actions/libraryActions';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { NeedAuthorization } from 'app/Auth';
+import { I18NLink, I18NMenu, t } from 'app/I18N';
+import { processFilters, encodeSearch } from 'app/Library/actions/libraryActions';
 
 export class Menu extends Component {
-
   libraryUrl() {
     const params = processFilters(this.props.librarySearch, this.props.libraryFilters.toJS());
-    return '/library/' + encodeSearch(params);
+    return `/library/${encodeSearch(params)}`;
   }
 
   uploadsUrl() {
     const params = processFilters(this.props.uploadsSearch, this.props.uploadsFilters.toJS());
-    return '/uploads/' + encodeSearch(params);
+    return `/uploads/${encodeSearch(params)}`;
   }
 
   render() {
-    const {links} = this.props;
+    const { links } = this.props;
     const user = this.props.user.toJS();
 
     const navLinks = links.map((link) => {
-      if (link.get('url').startsWith('http')) {
+      if (link.get('url') && link.get('url').startsWith('http')) {
```

Comment:
You could do const url = link.get('url') || '/'; so from that point you can always asume that the url is defined